src/utilities/typography.scss

```diff
@@ -190,9 +190,9 @@
 // Text styles
 /* Set the font weight to normal */
 .text-normal { font-weight: $font-weight-normal !important; }
-/* Set the font weight to bold */
-.text-bold { font-weight: $font-weight-bold !important; }
```

Comment:
Just to confirm, we will replace all `.text-bold` in dotcom with `.text-semibold`?

Reply:
@langermank Yesss, that should produce no visual diffs 😌

Reply:
I think we should leave this utility class here for now, but add it to the deprecated classes list. `deprecations.json` To give us time to move the uses on dotcom away from this class. But we should also change change it to the semibold variable.

```diff
- .text-bold { font-weight: $font-weight-bold !important; }
+ .text-bold { font-weight: $font-weight-semibold !important; }
```

Doing this also give PVC time to migrate the system argument which is [`font_weight: :bold`](https://primer.style/view-components/system-arguments#typography)